modules/document_repository/ajax/FileUpload.php

```diff
@@ -0,0 +1,387 @@
+<?php
+/**
+ * Media uploader.
+ *
+ * Handles media upload and update actions received from a front-end ajax call
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Media
+ * @author   Alex I. <ailea.mcin@gmail.com>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action == "getData") {
+        viewData();
+    } else if ($action == "upload") {
+        uploadFile();
+    } else if ($action == "edit") {
+        editFile();
+    } else if ($action == "getCategory") {
+        viewCategory();
+    } else if ($action == "uploadCategory") {
+        uploadCategory();
+    }
+}
+
+/**
+ * Handles the media update/edit process
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editFile()
+{
+    $db   =& Database::singleton();
```

Comment:
I think it should be `\Database` and `\User`

Reply:
done,thanks.